### ISSUE TITLE
treefacet question w/ possible fix for export

### ISSUE THREAD

**Issue by Thell** [rstudio/ggvis#419]
tl;dr - treefacet isn't known to vega and use of facet seems to work just fine with the ggvis tests and a cookbook visual check. Granted these are simplistic so is there a reason treefacet is being used rather than the known facet?

* * *

 The other day I ran into an issue with export_png failing when browser viewing worked just fine; it turned out to be the use of `label` instead of `labels` within my axis props. A waste of a time trying to run that down with vg2png's unhelpful 'unknown' but it told me that there is some possible argument matching being done that doesn't get done with export.

That got me to thinking about the reported issue with export_png and grouped data reported in [this long ggvis group post](https://groups.google.com/forum/#!searchin/ggvis/export_png/ggvis/eQ4XHmRnCCc/G7hwcR4vvh4J) and on [this SO question](http://stackoverflow.com/questions/27674431/exporting-ggvis-plot-with-grouped-data) and in ggvis issue https://github.com/rstudio/ggvis/issues/332.

A little checking resulted in the realization that while `treefacet` works fine when `ggvis.js` is available it is just plainly __not__ list in vega's [data manipulation transforms](https://github.com/trifacta/vega/wiki/Data-Transforms#data-manipulation-transforms). After making a [branch that alters the instances of `treefacet` to `facet`](https://github.com/Thell/ggvis/tree/thell/fix-export-of-group-transformed) and testing that the ggvis package works as well as the `export_png` I'm left wondering if there is a reason for that name choice that isn't apparent.

**Comment by Thell** [comment]
Here's an RPub image link showing it works...

[![ggvisexport](https://cloud.githubusercontent.com/assets/2185020/8384161/ccb8bb80-1c04-11e5-9788-bd920eb87890.png)](http://rpubs.com/Thell/90155)
